src/pages/Upsolve/CarouselUpsolve.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { Row, Col } from 'antd';
+import Carousel from 'react-multi-carousel'
+import "react-multi-carousel/lib/styles.css";
```

Comment:
All the css imports should be written below the Js imports.

Reply:
done